public/components/datasources/components/manage/accelerations/create_accelerations_flyout/selectors/source_selector.tsx

```diff
@@ -74,7 +74,11 @@ export const AccelerationDataSourceSelector = ({
   const loadDataSource = () => {
     setLoadingComboBoxes({ ...loadingComboBoxes, dataSource: true });
     http
-      .get(DATACONNECTIONS_BASE + `/dataSourceMDSId=${dataSourceMDSId}`)
+      .get(
+        dataSourceMDSId
+          ? `${DATACONNECTIONS_BASE}/dataSourceMDSId=${dataSourceMDSId}`
```

Comment:
Question: why does this endpoint have `/dataSourceMDSId=${dataSourceMDSId}` whereas for others we keep ` /${dataSourceMDSId ?? ''}` the mdsId as a path param in itself?